client/src/components/core/ms-modal/MsAlertModal.vue

```diff
@@ -15,9 +15,18 @@
   />
 </template>
 
+<script lang="ts">
+export interface MsAlertModalConfig {
+  title: string;
+  theme: MsReportTheme;
+  message: string;
+}
+</script>
```

Comment:
this is a rebase error, it should not be here but imported from types.ts instead.

Reply:
Done